worker.py

```
@@ -5,14 +5,122 @@
+
+**Issue Description:**
+{issue_body or 'No description provided.'}
+
+Respond with a numbered markdown list of implementation steps."""
+        client = genai.Client(api_key=api_key)
+        response = client.models.generate_content(
+            model="gemini-2.0-flash",
+            contents=prompt
+        )
+        return f"## 🗺️ AI-Generated Implementation Plan\n\n{response.text}"
+    except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)